### DISCUSSION TITLE
Display of two toggled buttons

### DISCUSSION THREAD

**Discussion by drwelby** [Textualize/textual#4198] [Q&A]
```
from textual.app import App, ComposeResult
from textual.widgets import Button


class MyApp(App):

    def on_button_pressed(self, event: Button.Pressed) -> None:
        event.button.disabled = True
        if event.button.id == "a":
            self.query_one('#b', Button).disabled = False
        elif event.button.id == "b":
            self.query_one('#a', Button).disabled = False

    def compose(self) -> ComposeResult:
        yield Button('A', variant='primary', id='a')
        yield Button('B', variant='primary', id='b', disabled=True)

if __name__ == '__main__':
    MyApp().run()
```
If I make two buttons that toggle each other's state, when I press one button the other activates, but seems to be in the darker blue "hover" color. If you then mouse on and off the button the button switches to the unhovered brighter blue color.

It seems otherwise functional so is there something else I need to do to get the enabled button to look right?

**Comment by TomJGooding** [comment]
This looks like an example of this issue?

- https://github.com/Textualize/textual/issues/3130

**Reply by drwelby** [reply]
Indeed! Not sure how I missed that searching the issues.
